### ISSUE TITLE
[Broken Website] PNGs used for diagrams on Wikipedia are unreadable when Dark Reader is active

### ISSUE THREAD

**Issue by artisticfox8** [darkreader/darkreader#11205]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Images explaining reactions are <img> tag transparent background png files. If Dark Reader is active, the background changes from white, and the png gets that dark grey color as it's background, making it unreadable. Screenshot attached.

### Website Address

https://en.m.wikipedia.org/wiki/Substitution_reaction

### Steps To Reproduce

1) Turn Dark Reader on
2) Go to Wikipedia
3) Select any chemistry article, like https://en.m.wikipedia.org/wiki/Substitution_reaction 
4) Look for images explaining reaction

### Screenshots

![Screenshot_20230430-233505_Firefox](https://user-images.githubusercontent.com/77014769/235377750-c237c3ba-6b5d-418b-bb36-37a036dfbe76.jpg)

### Operating System

Windows and Android

### Web Browser name and version

Firefox 112

### Dark Reader version

4.9.63

**Comment by artisticfox8** [comment]
I found a fix for the Wikipedia website.
`CSS
img{
   background: white;
}`
The fix does nothing to pictures which are not transparent, and makes the diagrams readable again.

**Comment by Drakeland** [comment]
It might be more accurate to select by the file extension. These appear to all be PNGs converted from SVGs, thus having a filename ending in `.svg.png`.

Additionally, following the method of fixing the display of math formulas in [#1805](https://github.com/darkreader/darkreader/pull/1805), we can invert the image contents to get appropriate dark mode colors.

Putting these findings together, the following CSS rule has worked for me on Chrome in the developer tools:

`img[src$=".svg.png"] {
  filter: invert(1);
}`

I'm currently not well practiced in forking and pull-requesting or I would add this fix myself.

**Comment by shellderr** [comment]
I agree this is important. For an article like [this](https://en.wikipedia.org/wiki/Coxeter%E2%80%93Dynkin_diagram) you have to turn dark reader off.
